config/default.yml

```diff
@@ -1,5 +1,3 @@
 inherit_from:
   - upstream.yml
-  - disable_all.yml
-  - cookstyle_base.yml
```

Comment:
this whack shit was meant to freeze the enable/disable status so that new cops wouldn't be enabled by default, but we wouldn't turn off anything that was already turned on in cookstyle.

so cookstyle_base.yml plus cookstyle.yml was the real config you were using.  but cookstyle_base was all the enabled-by-default cops in the old system which was mechanically generated.  and then cookstyle.yml was only the stuff that had been human-tweaked.

so, if all these changed worked the way you think they do in this PR then what you'd wind up with would be *only* the cops in cookstyle.yml running and everything else suddenly getting shut off, which is probably not what you want.

since `DisabledByDefault: true` doesn't seem to work and you've ripped out disable_all then you're going to get all that back -- but then additionally just get every new cop that rubocop ships as being enabled.

honestly, that might be what you want to do anyway, since you can just audit new cops once a year and figure out what is stupid or not.  in which case you should just delete the `DisabledByDefault: true` line and move on.

OTOH if you really want that to continue to work, i think it doesn't.

Reply:
(and nevermind...)